render_config.sh

```diff
@@ -63,5 +63,5 @@ $VAULT_COMMAND -field=data -format=json "secret/dsde/firecloud/$ENV/common/firec
 if [ $ENV == perf ]; then
   $VAULT_COMMAND -field=key "$COMMON_VAULT_PATH/testrunner/testrunner-sa" | base64 -d > "$INTEGRATION_OUTPUT_LOCATION/testrunner-sa.json"
   else
-    $VAULT_COMMAND -field=key "$ECM_VAULT_PATH/app-sa" | base64 -d > "$SERVICE_OUTPUT_LOCATION/ecm-sa.json"
+    $VAULT_COMMAND -field=key -format=table "$ECM_VAULT_PATH/app-sa" | base64 -d > "$SERVICE_OUTPUT_LOCATION/ecm-sa.json"
```

Comment:
Later versions of the `vault` binary use `-format=json` as the default. `-format=json` includes quotes around the output value, and the quotes cause baseball 64 decoding errors. Using `-format=table` omits the quotes.